scripts/e2e-tests.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# For running integration tests in travis.  This should happen any time we merge to staging.
+npm run webpack-prod;
+npm run serve &
+sleep 3;
```

Comment:
eek. what about writing a node.js script that loads the server.js as a module and watches for a start signal or something?

As is, this is just asking for flaky runs on travis over time.